crawler/parlamentares/frentes/fetcher_frentes.R

```diff
@@ -0,0 +1,25 @@
+#' @title Recupera frentes do Congresso
+#' @description A partir do id e da casa, recupera dados de uma frente
+#' @param id_prop ID de uma frente
+#' @param casa Camara ou senado
+#' @return Dataframe contendo informações de uma frente
+#' @examples
+#' frente <- fetch_frente(54012, "camara")
+fetch_frente <- function(id_frente, casa) {
+  print(paste0("Baixando informações da frente de id ", id_frente, "..."))
+  frente <- rcongresso::fetch_frentes(id_frente, casa)
```

Comment:
Não consegui executar pela seguinte mensagem de erro:
`Erro: 'fetch_frentes' não é um objeto exportado de 'namespace:rcongresso'`
Acredito que só poderei revisar quando o PR https://github.com/analytics-ufcg/rcongresso/pull/158 for aceito.